bot/internal/bot/label.go

```diff
@@ -73,6 +85,23 @@ func (b *Bot) labels(ctx context.Context, files []github.PullRequestFile) ([]str
 			continue
 		}
 
+		docs, _, err := classifyChanges(b.c.Environment, files)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
+
+		if docs {
+			labels = append(labels, "documentation")
```

Comment:
I would keep the original logic that attaches "documentation" label based on prefix and instead of calling "classifyChanges" above again, just looked at attached labels after the "for" loop below and added backport labels if "documentation" label was added by the prefix logic.